### ISSUE TITLE
Feature Request: Make short words entirely bold

### ISSUE THREAD

**Issue by Darpyface** [ansh/bionic-reading#91]
Short 2 letter words should be entirely highlighted. Words like "it", "is" or "as". You have to switch to reading non-bold text on these words, which is jarring as you are switching back and forth, and slows reading down.

**Comment by asieduernest12** [comment]
Thank you @Darpyface we will try and get this taken care of

**Comment by Julioevm** [comment]
I have to disagree on this, personally I prefer short words to remain with just one of the letters bold. This sort of stuff should be user configurable, because not everyone is going to have the same preferences.

**Comment by Julioevm** [comment]
This is very subjective. For some people it works, and notice a improvement in their reading speed (my case) and for othersit doesn't, it even makes it harder for them. Not sure if there have been studies, though.

**Comment by asieduernest12** [comment]
@magnum6actual [is a link to Renato (inventor) of bionic reading and his explanation of how he discovered it](https://www.youtube.com/watch?v=CRzMGjDmpuk) . And I have included some links about Hierarchy in visual design. Hope these help you get close to an understanding.


My take is that bionic reading falls under graphic design especially hierarchy and you can look it up. As bionic reading probably falls under a new aspect of visual hierarchy where we want the focus the readers gaze to only parts of word so that their brain fills it up.

Bellow are some info on visual hierarchy for `designers`. 
Bionic reading is an innovation where we are applying a know common technique in a totally new way.

Things to also note is that eyes do not move in a smooth continuous fashion (human eyes) unless they are tracking something that is moving

https://www.interaction-design.org/literature/topics/visual-hierarchy
![Screenshot from 2022-05-30 11-04-15](https://user-images.githubusercontent.com/20851930/171019505-dbb43506-66a5-4ea4-a9fb-bf30ec8a3042.png)

![Screenshot from 2022-05-30 11-04-46](https://user-images.githubusercontent.com/20851930/171019638-199a0d72-e655-4716-869f-513d672553e4.png)

**Comment by khinshankhan** [comment]
How about a compromise and creating a settings option to completely bold words of <= n length or else apply the 'algorithm'? Couple that with a default setting to retain current behavior 🤔

**Comment by Darpyface** [comment]
It might be better just to always bold the first two letters of any word. Words like “the” or “and” only have the first letter bolded but you cannot determine what they mean without the second letter.

**Comment by ansh** [comment]
This a subjective matter and we should give users toggles to be able to do this. There is no "right" way to do it as this is not a scientifically backed method.

**Comment by auryn31** [comment]
> This a subjective matter and we should give users toggles to be able to do this. There is no "right" way to do it as this is not a scientifically backed method.

Completely agree, but if you take a look at the founders algorithm, there is a "right" way. He says, if the word is one char long, it is bold if its two, only the first char. If its three, its the first 2 and on 4 its also the first two. 

Here is my [elm implementation](https://github.com/auryn31/bionic-reading-elm/blob/60dada993cf18bc5cf6c8f437c67e6074ab1642d/src/Main.elm#L125)


But i think giving the choice is the best way. :-)


Links to the patent:

https://depatisnet.dpma.de/DepatisNet/depatisnet?window=1&space=main&content=familie&action=treffer&docid=DE102017112916A1&firstdoc=1

https://worldwide.espacenet.com/maximizedOriginalDocument?ND=4&flavour=maximizedPlainPage&locale=en_EP&FT=D&date=20171214&CC=US&NR=2017358238A1&KC=A1
